terraform/services/bucket-access-log/README.md

```diff
@@ -0,0 +1,12 @@
+# Terraform for bucket-access-log in target accounts
```

Comment:
Your naming below convinced me that we should go with the plural "bucket-access-logs" for the name of the service instead. Please rename, sorry!